myokit/_sim/cvodessim.py

```diff
@@ -121,9 +121,13 @@ class Simulation(myokit.CModule):
 
     ``model``
         The model to simulate
-    ``protocol``
-        An optional :class:`myokit.Protocol` to use as input for variables
-        bound to ``pace``.
+    ``protocols``
+        An optional dict mapping paced variable labels (str) to a
+        :class:`myokit.Protocol` or None. if None, the paced variable will be
+        set to 0, or the pacing protocol can be set later using
+        :meth:set_protocol. For backwards compatibility, a single protocol
+        of type :class:`myokit.Protocol` can be passed in instead of a dict,
+        the label in this case will be 'pace'.
```

Comment:
I imagine the single or no protocol cases will remain more popular than the multi-protocol sims, so probably better to phrase as e.g. "A protocol, a dict mapping ..., or None. The protocol can be..."

Reply:
yup, will do